scripts/windows-bundler.sh

```diff
@@ -73,14 +74,20 @@ cp "resources/Tachidesk Browser Launcher.bat" $release_name
 cp "resources/Tachidesk Debug Launcher.bat" $release_name
 cp "resources/Tachidesk Electron Launcher.bat" $release_name
 
-zip_name=$release_name.zip
-zip -9 -r $zip_name $release_name
```

Comment:
keep the zip release, we could probably convert it into a standalone version some time?